### ISSUE TITLE
[Object Map Iterator] in Tabs

### ISSUE THREAD

**Issue by alexpchin** [PedroBern/react-native-collapsible-tab-view#101]
Hi @PedroBern @andreialecu 

When upgrading to `4.0.0-next.0` and running the basic version in a react-native application (not expo) as per:

```ts
import { Tabs } from 'react-native-collapsible-tab-view';
.
.
.

return (
  <Tabs.Container HeaderComponent={_renderHeader}>
      <Tabs.Tab name="A">
        <Tabs.FlatList
          data={[0, 1, 2, 3, 4]}
          keyExtractor={(v) => v + ''}
          renderItem={null}
        />
      </Tabs.Tab>
      <Tabs.Tab name="B">
        <Tabs.ScrollView>{null}</Tabs.ScrollView>
      </Tabs.Tab>
 </Tabs.Container>
)
```

I see [Object Map Iterator], where the tabs are meant to be:

<imc src="https://user-images.githubusercontent.com/40461/107409517-2fbeb280-6b04-11eb-8d0e-8354d9bc75a2.png" width="300px" />

I also see:

<imc src="https://user-images.githubusercontent.com/40461/107409729-6399d800-6b04-11eb-98a1-d04215560793.png" width="300px" />

I'm also having a problem scrolling the header. I will add additional notes here as I investigate.

**Comment by andreialecu** [comment]
I created a bare rn app here and cannot reproduce this:
https://github.com/andreialecu/rnctv-react-native-bare-test

![simulator_screenshot_D86B66F8-CBB1-42F3-AD25-EAAD071BC2D9](https://user-images.githubusercontent.com/697707/107497243-ff762300-6b9a-11eb-8ea4-93f815c456a5.png)

What might be different between the bare repro above and yours?

**Comment by alexpchin** [comment]
I also do this in PR #101 and it also seemed to be ok for me, I'm trying to get to the bottom of the problem. It's strange because it happens on a screen in that just implements the QuickStart example. I will fork your repo and perhaps add the packages that I am using and update this issue if I find anything.

**Comment by andreialecu** [comment]
Might be related to babel. Do you use any other babel (or metro) plugins?

**Comment by andreialecu** [comment]
> I'm also having a problem scrolling the header. I will add additional notes here as I investigate.

@alexpchin I just realised on the bare rn app that using `2.0.0-rc.3` of `reanimated` breaks pretty much everything. Please try going back to `rc.0` and testing things with that version.

Expo only supports `rc.0` and I think @PedroBern wants to mainly follow Expo support.

**Comment by alexpchin** [comment]
Hi @andreialecu I've found out what the problem is, I think last night @PedroBern released the latest version as "4.0.0-next.0", in my package JSON. (Please see the original issue post?) I copied from the Release Tags very soon after he had pushed, so I think `4.0.0-next.0` changed to `4.0.0-rc.0`?

```
"react-native-collapsible-tab-view": "4.0.0-next.0",
```

I have now changed to:

```
"react-native-collapsible-tab-view": "^4.0.0-rc.0"
``` 

Sorry! Really strange how that change I made to hooks seemed to fix things... Really threw me. I will close this now and then move onto the next issue!
